src/app/global/notifications/NotificationItem.jsx

```diff
@@ -31,11 +31,12 @@ class NotificationItem extends Component {
                     this.props.type === "warning" ? "notifications__item--warning" : ""
                 } ${this.props.type === "positive" ? "notifications__item--positive" : ""}`}
             >
-                {this.props.message}
+                <p className="notifications__message">{this.props.message}</p>
```

Comment:
I can't find `notifications__message` class being used?

Reply:
<p> was left in by mistake, now removed.